unit-tests/func/frames/color_frame_frops_test.py

```diff
@@ -0,0 +1,52 @@
+from __future__ import division
+import logging
+import time
+
+import pyrealsense2 as rs
+
```

Comment:
Add pragma D400 for this test to run with LibCI - should it run also with L500? SR300? Please check